Some weights of RobertaForSequenceClassification were not initialized from the model checkpoint at roberta-large and are newly initialized: ['classifier.dense.bias', 'classifier.dense.weight', 'classifier.out_proj.weight', 'classifier.out_proj.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.


Map:  69%|██████████████████████████████████▎               | 30000/43780 [00:03<00:01, 8349.26 examples/s]
  0%|                                                                             | 0/4000 [00:00<?, ?it/s]/home/nlp-04/anaconda3/envs/fuck/lib/python3.11/site-packages/torch/nn/parallel/_functions.py:68: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '
























































































































































































































































































 10%|██████▌                                                          | 400/4000 [09:33<1:25:57,  1.43s/it]






  File "/data/2_data_server/nlp-04/lost_technology/test.py", line 104, in <module> [00:13<00:00,  2.11it/s]
    trainer.train()
  File "/home/nlp-04/anaconda3/envs/fuck/lib/python3.11/site-packages/transformers/trainer.py", line 1555, in train
    return inner_training_loop(
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/nlp-04/anaconda3/envs/fuck/lib/python3.11/site-packages/transformers/trainer.py", line 1929, in _inner_training_loop
    self._maybe_log_save_evaluate(tr_loss, model, trial, epoch, ignore_keys_for_eval)
  File "/home/nlp-04/anaconda3/envs/fuck/lib/python3.11/site-packages/transformers/trainer.py", line 2256, in _maybe_log_save_evaluate
    metrics = self.evaluate(ignore_keys=ignore_keys_for_eval)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nlp-04/anaconda3/envs/fuck/lib/python3.11/site-packages/transformers/trainer.py", line 2972, in evaluate
    output = eval_loop(
             ^^^^^^^^^^
  File "/home/nlp-04/anaconda3/envs/fuck/lib/python3.11/site-packages/transformers/trainer.py", line 3261, in evaluation_loop
    metrics = self.compute_metrics(EvalPrediction(predictions=all_preds, label_ids=all_labels))
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/2_data_server/nlp-04/lost_technology/test.py", line 51, in compute_metrics
    softmax_preds = torch.softmax(torch.from_numpy(preds), dim=-1).numpy()
                    ^^^^^
NameError: name 'torch' is not defined